content/downloads.md

```diff
@@ -0,0 +1,10 @@
+---
+date: 2014-03-10
+title: Download
+image: 'speakers.jpg'
+---
+[Install libvarnam](../post/install-libvarnam)
```

Comment:
Can you please replace this with GoVarnam ? We now have binary downloads (Linux 64 bit) : https://github.com/varnamproject/govarnam/releases/tag/v1.4.0

Also add Windows - Coming Soon
Mac - Coming Soon

Web Demo - /editor (https://varnamproject.github.io/editor/)